gen/definitions/classes.yaml

```diff
@@ -9,6 +9,37 @@ fvTenant:
   resource_name: "tenant"
   resource_identifier: "tn"
 
+fvTrackList:
+  resource_name: "ip_sla_track_list"
+  ui_locations:
+    - "Tenants -> Policies -> Protocol -> IP SLA -> Track Lists"
+  sub_category: "Tenant Policies"
+
+fvRsOtmListMember:
+  resource_name: "relation_to_ip_sla_track_member"
+  ui_locations:
+    - "Tenants -> Policies -> Protocol -> IP SLA -> Track Lists -> Track list to track member"
+  sub_category: "Tenant Policies"
+
+fvTrackMember:
+  resource_name: "ip_sla_track_member"
+  ui_locations:
+    - "Tenants -> Policies -> Protocol -> IP SLA -> Track Members"
+  sub_category: "Tenant Policies"
+
+fvRsIpslaMonPol:
+  resource_name: "relation_to_ip_sla_monitoring_policy"
+  required_as_child: true
```

Comment:
Why are we declaring this here? Can we deduce this from the meta files?

Reply:
because in UI it is required, not in meta so requires a overwrite to make it match UI